### ISSUE TITLE
Broken pipe errors

### ISSUE THREAD

**Issue by sgtg-us** [ztgrace/changeme#60]
[Broken Pipe errors.docx](https://github.com/ztgrace/changeme/files/1439757/Broken.Pipe.errors.docx)
I'm able to get changeme to work when using the IP directly or using a file with less than 5 IP addresses. When I try to run it against a file with several addresses or against a subnet, it comes back with No default credentials and a bunch of "IOError: [Errno 32] Broken pipe" errors. Attached are screenshots showing the results from 4 different tests.

**Comment by ztgrace** [comment]
Thanks for the bug report. The preferred configuration is to use Redis as the queue manager. I've had nothing but issues with the in-memory queues.

Until I can get around to fixing this, please use Redis.

**Comment by sgtg-us** [comment]
Will do. thanks!

**Comment by AdrianVollmer** [comment]
How do I use Redis? I also get lots of broken pipe exceptions: 

```
Process Process-5:
Traceback (most recent call last):
  File "usr/lib/python2.7/multiprocessing/process.py", line 267, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/home/avollmer/git/changeme/changeme/scan_engine.py", line 107, in fingerprint_targets
    while self.fingerprints.qsize() != 0:
  File "<string>", line 2, in qsize
  File "/usr/lib/python2.7/multiprocessing/managers.py", line 758, in _callmethod
    conn.send((self._id, methodname, args, kwds))
IOError: [Errno 32] Broken pipe

```

**Comment by sgtg-us** [comment]
I can't figure out how to use changeme with redis either. Is there a changeme command to use redis for the queue? I've tried using redis-server and redis-cli without luck. Running changeme in your ztgrace/changeme docker container defaults to in-memory queue.

**Comment by ztgrace** [comment]
This should be fixed with v1.1.
